testsuite/features/step_definitions/command_steps.rb

```diff
@@ -661,6 +665,11 @@
   raise "Command Output #{@command_output} don't include #{arg1}" unless @command_output.include? arg1
 end
 
+When(/^I (start|stop) "([^"]*)" service on "([^"]*)"$/) do |action, service, host|
```

Comment:
Don't we already have this somewhere ?

Reply:
Right! I coded this before the PR from Vanda that also required that step, so it must be already somewhere! Thanks for remind me it!

Reply:
Similar, but not the same, what Vanda included some time ago were these steps:
```
Then(/^service "([^"]*)" is enabled on "([^"]*)"$/) do |service, host|
  node = get_target(host)
  output, _code = node.run("systemctl is-enabled '#{service}'", check_errors: false)
  output = output.split(/\n+/)[-1]
  raise "Service #{service} not enabled" if output != 'enabled'
end

Then(/^service "([^"]*)" is active on "([^"]*)"$/) do |service, host|
  node = get_target(host)
  output, _code = node.run("systemctl is-active '#{service}'", check_errors: false)
  output = output.split(/\n+/)[-1]
  raise "Service #{service} not active" if output != 'active'
end
```
But those are only looking for a status, while this step is actually changing the status, by starting or stopping a service passed by parameter.